onnxruntime/core/graph/contrib_ops/contrib_defs.cc

```diff
@@ -2708,6 +2708,10 @@ This op functions in much the same was as Dropout-11 and Dropout-13 do, execpt t
     RegisterNchwcSchemas();
   }
 #endif
+
+#ifdef USE_MPI
```

Comment:
Should this be both USE_MPI && USE_NCCL given that all these collectives are implemented using NCCL?

Reply:
this is the schema part, not the kernels. we will need it one day another EP implement the collectives. So i don't think we should guard it with NCCL flag.